src/components/CustomCheckbox.tsx

```diff
@@ -0,0 +1,55 @@
+import { Field } from 'formik'
+import { CorrectAnswerIcon, WrongAnswerIcon } from './Icons'
+interface CheckboxProps {
+  label: string
+  type: string
+  correct: boolean
+  disabled: boolean
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  showAnswer: boolean
+  selected: boolean
+}
+
+export const Checkbox = ({
```

Comment:
Minor: I think we discussed the issue / challenge of coming up with a name for this that maintains clarity early on. My only comment with the current version is it is somewhat inconsistent relative to our other files where the component and filename match (the new `Icons.tsx` is an exception, but that makes natural sense due to it including multiple components).

Thoughts on a potential alternative name in case y'all want to rename it altogether:
* `Selectable`
* `SelectableOption`
* `SelectableChoice`
* `SelectableBox`
* `SelectableAnswer`
* `FieldBox`
* `FormSelectable`
* `AnswerBox`

Reply:
I like `SelectableAnswer` or `FormSelectable` - leaning more towards `FormSelectable`.

Reply:
Updated per this [commit](https://github.com/openstax/k12-apps-raise/pull/89/commits/040f8931d9c2d6451b0d23a1920bccde6a4843cd).